features/infrastructure/cluster-capacity.feature

```diff
@@ -4,7 +4,7 @@ Feature: cluster-capacity related features
   # @author weinliu@redhat.com
   # @case_id OCP-14799
   @admin
-  @flaky
+  @inactive
```

Comment:
`flaky` for short term, while `inactive` for long term.
Based on the description, sounds like you still need the test, just wan to disable it for a short time.

Reply:
yes, `flaky` will have the case go to waiting state in the test run, and the team got pinged frequently, so prefer to have it being `inactive` for a while

Reply:
Change to inactive in code does not resolve the issue, as test case selection is done via interacting with Polarion.
If you would like the test case not picking up in Jenkins CI for some time, it's suggested to move the test case into "not automated" status in Polarion.